specs/Deployment/DeploymentAPI.md

```diff
@@ -3,101 +3,117 @@
 This is the spec for proposal
 [Windows App Runtime Deployment API #1240](https://github.com/microsoft/WindowsAppSDK/issues/1240).
 
-The Windows App SDK (WinAppSDK) for packaged apps involves the framework, main, and singleton
-packages, which are all signed and published by Microsoft. The framework package contains the
-WinAppSDK binaries used at run time, and it is installed with the application. The main package
-contains out-of-process services that are brokered between apps, such as push notifications, and it
-is also needed for the framework to be serviced by the Microsoft Store. The singleton package
-supports a single long-running process that is brokered between apps for features like push
-notifications.
-
-Currently, packaged apps can only declare dependencies on WinAppSDK framework package. In order to
-get the other WinAppSDK packages (main, singleton) deployed, the packaged apps can use the mechanism
+The Windows App SDK (WinAppSDK) for packaged apps involves the Framework, Main, and Singleton
+packages, which are all signed and published by Microsoft.
+
+Currently, packaged apps can only declare dependencies on WinAppSDK Framework package. In order to
+get the other WinAppSDK packages (Main, Singleton) deployed, the packaged apps can use the mechanism
 described here as part of their first-run experience.
 
 > **Note that in Windows App SDK version 1.0**, only unpackaged apps that are full trust or MSIX
 > packaged apps that have the packageManagement restricted capability have the permission to use the
-> Deployment API to install the main and singleton package dependencies. Support for partial trust
-> MSIX packaged apps will be coming in later releases. Additionally, support for main and singleton
+> Deployment API to install the Main and Singleton package dependencies. Support for partial trust
+> MSIX packaged apps will be coming in later releases. Additionally, support for Main and Singleton
 > package deployment through the Microsoft Store will be coming in later releases.
 
 # Description
 
 When a packaged app using the WinAppSDK is deployed from the Store, it will be installed with the
-application's main package and the WinAppSDK framework package. The WinAppSDK main and singleton
+application's Main package and the WinAppSDK Framework package. The WinAppSDK Main and Singleton
 packages may not already be present on the system, or may not be at the version required by the
-application. The Deployment API enables a developer to check when the required WinAppSDK main and
-singleton packages are missing and get them installed.
+application. The Deployment API enables a developer to check when the required WinAppSDK Main and
+Singleton packages are missing and get them installed.
```

Comment:
Reading through this, it sounds like all of the sample code shown in this spec should be included in all apps?

Reply:
Yes, in all Packaged Apps.